recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-    number: 0
+    number: 2
```

Comment:
Build numbers are bump when the same version had a functional recipe change.

The only recipe change here is the checksum, which should not change from the same version, unless upstream changed the version in-place, which is terrible for reproducibilty.

Also, going from `0` to `2` make bots and people who track the project wondering what happened to build number `1`. That breaks mirrors and people workflows.